src/game/WorldStateExtensions.ts

```diff
@@ -43,6 +43,31 @@ export function worldStateCycle(
     }
 }
 
+/**
+ * Creates a reducer for world state parameters
+ * 
+ * @param targetId The id of the state parameter that will receive the result
```

Comment:
Good documentation. This will be helpful when we revisit this code in a few months!